src/pages/Graph/GraphHelpFind.tsx

```diff
@@ -322,9 +324,12 @@ export default class GraphHelpFind extends React.Component<GraphHelpFindProps> {
                     <Table.Body
                       rowKey="id"
                       rows={[
-                        { id: 'ec00', c: 'http <op> <number>', n: 'unit: requests per second' },
-                        { id: 'ec10', c: '%err <op> <number>', n: 'range: [0..100]' },
-                        { id: 'ec20', c: '%traffic <op> <number>', n: 'range: [0..100]' },
+                        { id: 'ec00', c: 'grpc <op> <number>', n: 'unit: requests per second' },
+                        { id: 'ec10', c: '%grpcerr <op> <number>', n: 'range: [0..100]' },
+                        { id: 'ec20', c: '%grpctraffic <op> <number>', n: 'range: [0..100]' },
+                        { id: 'ec23', c: 'http <op> <number>', n: 'unit: requests per second' },
+                        { id: 'ec24', c: '%httperr <op> <number>', n: 'range: [0..100]' },
+                        { id: 'ec25', c: '%httptraffic <op> <number>', n: 'range: [0..100]' },
```

Comment:
In the "Examples" a change from `%traffic` -> `%httptraffic` is missing

Reply:
Si, gracias.